build/Linux+BSD/org.musescore.MuseScore.desktop.in

```diff
@@ -12,9 +12,9 @@ Comment=Create, play and print beautiful sheet music
 Comment[ru]=Визуальный редактор нотных партитур
 Comment[fr]=Gravure de partitions musicales
 Icon=mscore@MUSESCORE_INSTALL_SUFFIX@
-Exec=mscore@MUSESCORE_INSTALL_SUFFIX@ %F
+Exec=mscore@MUSESCORE_INSTALL_SUFFIX@ %U
```

Comment:
:+1: This is working for me to open score URLs via the web browser and score files via the file manager on Ubuntu.